phpcs.xml.dist

```diff
@@ -34,5 +34,8 @@
   <exclude-pattern>*/node_modules</exclude-pattern>
   <exclude-pattern>*/vendor</exclude-pattern>
   <exclude-pattern>*/default.local.settings.php</exclude-pattern>
+  <!-- Drupal itself doesn't meet code standards. -->
+  <!-- @see https://www.drupal.org/project/drupal/issues/2571965 -->
+  <exclude-pattern>*/settings.php</exclude-pattern>
```

Comment:
Shouldn't this already take care of it? https://cgit.drupalcode.org/drupal/tree/sites/default/default.settings.php#n3

Reply:
Good catch, thanks. The problem is that that line was only added a little over a year ago. Projects created before then won't have it. I'm not sure we have a mechanism to update it but maybe we should.